pkg/platform/packet/template.go

```diff
@@ -100,6 +100,14 @@ module "packet-{{.Config.ClusterName}}" {
   {{- end }}
 
   disable_self_hosted_kubelet = {{ .Config.DisableSelfHostedKubelet }}
+  
+  {{- if .Config.KubeAPIServerExtraFlags }}
+  kube_apiserver_extra_flags = [
+    {{- range $dep := .Config.KubeAPIServerExtraFlags }}
+    {{ $dep }},
+    {{- end }}
```

Comment:
I think we need quotes here? I'd also use dot notation for simpler syntax
```suggestion
    {{- range .Config.KubeAPIServerExtraFlags }}
    "{{ . }}",
    {{- end }}
```